api-admin/openapi.yaml

```
@@ -24,6 +24,21 @@ paths:
         500:
           description: Something goes wrong
 
+  "/applications.registration":
```

```suggestion
  "/applications.create":
```
Я думаю есть разница между регистрацией и созданием. Мне кажется будут конфликты именований, между регистрацией В приложении и регистрацией приложения